README.md

```diff
@@ -45,7 +45,7 @@ or try our example:
 ```
 rill init-example
 ```
-(Please note that the command `rill init-example` is temporarily unavailable on Windows.)
+<!-- (Please note that the command `rill init-example` is temporarily unavailable on Windows.) -->
```

Comment:
should we keep this for people on working versions using windows (I talked to one yesterday, I know they exist!)